### PR TITLE
Adding Plugin Tutorials to Docs

### PR DESCRIPTION
**Pull request by kgardnr**
## What does this PR do?
- Adds @okbel's awesome tutorial 🥇 
- Now includes Part 1 and Part 2

## Next Steps
- Add Parts 1 and 2 in Spanish

### CODE REVIEW

File: docs/_config.yml [hunk 1]
```diff
@@ -136,12 +136,20 @@ sidebar:
...
     - title: API
       children:
         - title: Server Plugins
           url: /reference/server/
         - title: GraphQL
           url: /reference/graphql/
+    - title: Migrating
+      children:
+        - title: Migrating to v4.0.0
+          url: /migration/4/
```
Comment (wyattjoh) [inline docs/_config.yml]: This was duplicated incorrectly.

File: docs/source/05-01-building-basic-plugin.md [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+
+## Reactions
+
+Talk exposes a friendly API to create new reactions. To explore the capabilities of Talk we are going to create a new reaction together step-by-step.
+
+In Talk, there are currently three ways commenters can react to comments: Like, Love, and Respect. These reactions are separated into plugins so that you can customize which reactions you want to use by toggling them on or off - or by adding your own custom reactions, which is what we are going to do today.
+
+We can create a new plugin from scratch or we can use the Talk CLI to generate a plugin template for us to use. CLI stands for Command Line Interface, meaning can access utilities via the command line that make it easy to interact and integrate with Talk.
+
+Please note this tutorial assumes you have already [installed and configured Talk locally](https://docs.coralproject.net).
```
Comment (wyattjoh) [inline docs/source/05-01-building-basic-plugin.md]: Should reference the specific page in the docs using a relative link:

```diff
-Please note this tutorial assumes you have already [installed and configured Talk locally](https://docs.coralproject.net).
+Please note this tutorial assumes you have already [installed and configured Talk locally](/talk/)
```

File: docs/source/05-01-building-basic-plugin.md [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+## Creating a new plugin using the Talk CLI
+
+* Open your terminal
+* Go to the Talk folder
+* Enter `./bin/cli-plugins` in the command line
+
+
+![Using the Plugin CLI](/talk/images/pride_reaction_tutorial_1.png)
+
+You will see 3 options: `create`, `list` y `reconcile`
```
Comment (wyattjoh) [inline docs/source/05-01-building-basic-plugin.md]: Misplaced `y`?

**Review by wyattjoh — CHANGES_REQUESTED** [review]
Few issues I noticed

File: docs/source/05-01-building-basic-plugin.md (hunk 3 above)
Comment (kgardnr) [reply docs/source/05-01-building-basic-plugin.md]: Jajaja ooooops

File: docs/_config.yml (hunk 1 above)
Comment (kgardnr) [reply docs/_config.yml]: Fixed!

File: docs/source/05-01-building-basic-plugin.md (hunk 2 above)
Comment (kgardnr) [reply docs/source/05-01-building-basic-plugin.md]: Fixed!

**Comment by kgardnr** [comment]
@wyattjoh fixed and ready for re-review!

**Review by wyattjoh — DISMISSED** [review]

File: docs/source/05-02-customizing-plugins-coral-ui.md [hunk 4]
```diff
@@ -0,0 +1,337 @@
...
+
+We will create the folder `assets` and place our two files inside it: `ColoredRainbowIcon.svg` and `GrayscaleRainbowIcon.svg`. We can export them both with Sketch or simply copy the SVG code into each file.
+
+## Using an SVG in our components
+
+We are going to import our SVG icons just as we did with our components, the only difference is the `.svg` at the end.
+
+```js
+import ColoredRainbowIcon from '../assets/ColoredRainbowIcon.svg';
+import GrayscaleRainbowIcon from '../assets/GrayscaleRainbowIcon.svg’;
```
Comment (wyattjoh) [inline docs/source/05-02-customizing-plugins-coral-ui.md]: Looks like the quote at the end is wrong here.

Comment (kgardnr) [reply docs/source/05-02-customizing-plugins-coral-ui.md]: Fixed!

**Comment by kgardnr** [comment]
@wyattjoh @okbel ready for final review!

**Review by wyattjoh — APPROVED** [review]

**Pull request merged**